spec/inquisition/collector_spec.rb

```diff
@@ -14,10 +14,39 @@
           def call
             ['a', ['b', 'c']]
           end
+
+          def self.enabled?
+            true
+          end
         end
       end
 
       it { is_expected.to match_array(['a', 'b', 'c']) }
     end
+
+    context 'call enabled runner' do
```

Comment:
`when the runner is disabled` instead of `call enabled runner`
https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ContextWording